tdrs-frontend/docker-compose.local.yml

```diff
@@ -1,10 +1,12 @@
-# Local version of Docker compose, using the Dockerfile.local instead and mounting node_modules
-version: "3"
+# Local version of Docker compose, using the localdev docker build target
+# instead and additionally mounts node_modules for faster builds
+version: "3.4"
 
 services:
   tdp-frontend:
     build:
       context: .
-      dockerfile: Dockerfile.local
+      target: localdev
```

Comment:
This `target` option allows us to stop the Dockerfile at a specific stage, which allows us to prevent nginx from running in the local context.